php/php-rr-sdk.md

```diff
@@ -281,11 +271,13 @@ command wrapped via dedicated class:
 ```php
 function run()
 {
-    yield new Activity('name', 'values', [/* options */]);
+    $activity = new Activity('name', 'values', [/* options */]);
```

Comment:
Is "new Activity" required by PHP? Why not 

$activity = Workflow::executeActivity('name', ...);
$result = yield $activity;

Reply:
Sure. It's still new Activity behind the hood.